flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/job/JobExceptionsHandlerTest.java

```diff
@@ -79,26 +113,35 @@ public void testGetJobExceptionsInfo() throws HandlerRequestException {
 
     private static void checkExceptionLimit(
             JobExceptionsHandler jobExceptionsHandler,
-            AccessExecutionGraph graph,
+            ExecutionGraphInfo graph,
             int maxNumExceptions,
             int numExpectedException)
             throws HandlerRequestException {
         final HandlerRequest<EmptyRequestBody, JobExceptionsMessageParameters> handlerRequest =
-                createRequest(graph.getJobID(), numExpectedException);
+                createRequest(graph.getJobId(), numExpectedException);
         final JobExceptionsInfo jobExceptionsInfo =
                 jobExceptionsHandler.handleRequest(handlerRequest, graph);
         final int numReportedException =
                 maxNumExceptions >= numExpectedException ? numExpectedException : maxNumExceptions;
         assertEquals(jobExceptionsInfo.getAllExceptions().size(), numReportedException);
     }
 
-    private static AccessExecutionGraph createAccessExecutionGraph(int numTasks) {
+    private static ExecutionGraphInfo createAccessExecutionGraph(int numTasks) {
         Map<JobVertexID, ArchivedExecutionJobVertex> tasks = new HashMap<>();
         for (int i = 0; i < numTasks; i++) {
             final JobVertexID jobVertexId = new JobVertexID();
             tasks.put(jobVertexId, createArchivedExecutionJobVertex(jobVertexId));
         }
-        return new ArchivedExecutionGraphBuilder().setTasks(tasks).build();
+
+        final SortedSet<TaskFailureHistoryEntry> exceptionHistory = new TreeSet<>();
+        exceptionHistory.add(
```

Comment:
This was added temporarily to have the test succeed again. It's going to be cleaned up as part of the test refactoring.